.eslintignore

```diff
@@ -1,8 +1,12 @@
 .git
 .github
+.husky
```

Comment:
edit the .eslintignore to cover the same files as the .prettierignore, and sort the lines alphabetically.